content/en/docs/reference/config/security/conditions/index.md

```diff
@@ -7,7 +7,7 @@ aliases:
 ---
 
 This page describes the supported keys and value formats you can use as conditions
-in the `when` field of [authorization policy resources](/docs/reference/config/security/authorization-policy/).
+in the `when` field of [authorization policy resources](/docs/reference/config/security/authorization-policy/#Condition).
```

Comment:
Are the changes in the `conditions/index.md` valid though? I guess they are not generated from the api, am I right?
If they are, I'll re-open the PR with those changes.